bin/http_server.re

```diff
@@ -161,6 +161,20 @@ let handle_receive_operation_gossip =
       Ok();
     },
   );
+
+let handle_trusted_validators_membership =
+  handle_request(
+    (module Trusted_validators_membership),
```

Comment:
I'm not sure if this name makes sense

Reply:
Happy to use a different one if you have a suggestion. It seems fine to me.

Reply:
Trusted_validators_membership -> rusted_validators_membership_change